### DISCUSSION TITLE
Anybody using this for gaming

### DISCUSSION THREAD

**Discussion by ghost** [debauchee/barrier#1268] [Q&A]
Actually performing surprisingly good in Valorant. I only use the keyboard forwarding part and put the mouse aside as the machine has another gaming mouse.

However I have noticed some issues in other game like Amid Evil where I cannot do some jumps feels lame but then I go to the host keyboard and it works right away.

Will be testing it on other games.

**Comment by jmdavis** [comment]
I've found it to be frequently the case that the mouse does not work properly with barrier (or with synergy before that, so it's not a new issue) when Windows is a client (I don't know about when it's the server). I actually had to get a USB switch to be able to switch my keyboard and mouse between my main desktop (currently FreeBSD) and my Windows desktop so that I could play games without having to physically unplug my mouse from one computer and plug it into the other.

**Comment by Akilan1999** [comment]
@jmdavis I had the same issue when testing out Minecraft. But in my case both the client and server where Linux Mint and Ubuntu.

**Comment by xuijuthub** [comment]
running and playing games for over 4 months this way, no struggles - worked well with all games tested.
server: 1:1 libinput driver linux with realtime kernel patches
client: ws2008r2 (it's like w7)
it seems to have less jitter and no perceivable delay perhaps even lower than windows default driver polling mechanism thing. but no measurements to (dis)prove:-(
